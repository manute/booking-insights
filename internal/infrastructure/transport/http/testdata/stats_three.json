[
  {
    "request_id":"bookata_XY123",
    "check_in":"2020-01-01",
    "nights":1,
    "selling_rate":50,
    "margin":20
  },
  {
    "request_id":"kayete_PP234",
    "check_in":"2020-01-04",
    "nights":1,
    "selling_rate":55,
    "margin":22
  },
  {
    "request_id":"trivoltio_ZX69",
    "check_in":"2020-01-07",
    "nights":1,
    "selling_rate":49,
    "margin":21
  }
]
