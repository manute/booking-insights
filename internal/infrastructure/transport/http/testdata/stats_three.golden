{"max_night":12.1,"min_night":10,"avg_night":10.796666666666667}